src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallationPageObject.java

```diff
@@ -45,4 +45,9 @@
      * Clients should not interact with this ToolInstalltion before updates was loaded.
      */
     String installer();
+
+    /**
+     *  Name of the concrete variant of the tool to install. IE Git or JGit
+     */
+    String variant() default "";
```

Comment:
I am not really familiar with git plugins, but the variant seems to be specific to `git-client` plugin rather than Jenkins tools installers.